src/utils/createCleanEvaluation.js

```diff
@@ -8,8 +8,10 @@ export function createCleanEvaluation() {
     evaluationData: {},
     meta: {},
   };
+  let principleApplicability = {};
 
   for (const principle of atag) {
+    principleApplicability[principle.principle.num] = true;
```

Comment:
for each principle in ATAG, update the empty object from above with the `num` of that principle and set to `true` (so we'll have `true` as the default applicability)